ansible/roles/test/files/tools/loganalyzer/loganalyzer_common_ignore.txt

```diff
@@ -29,6 +29,14 @@ r, ".* ERR kernel:.* Module gpio_ich is blacklisted.*"
 r, ".* skipping since it causes crash: SAI_STP_ATTR_BRIDGE_ID.*"
 r, ".* ERR monit.*Expected containers not running: telemetry.*"
 
+# Errors for config reload on broadcom platform on 202311
+r, ".* ERR swss#orchagent: :- queryHashNativeHashFieldListEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- queryHashNativeHashFieldListAttrCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- querySwitchEcmpHashAlgorithmEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR swss#orchagent: :- querySwitchLagHashAlgorithmEnumCapabilities: Failed to get attribute.*"
+r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:_brcm_sai_read_fec_stat_err_counters.* failed with error Feature unavailable.*"
+r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:brcm_sai_get_port_stats.* port fdr stats get failed with error Feature unavailable.*"
+
```

Comment:
@byu343 thank you for the update, but I still didn't find "port fdr stats get failed with error Feature unavailable" in our nightly test result. Probably, not related to sonic-net/SONiC#1101 as well. Can we remove it for now and add it back if we observe it in our nightly test and confirm that it's not image issue?

Reply:
Sure, I will create a separate PR for FEC counter error from broadcom SAI:
r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:_brcm_sai_read_fec_stat_err_counters.* failed with error Feature unavailable.*"                                                                                                                             
r, ".* ERR syncd\d*#syncd.*SAI_API_PORT:brcm_sai_get_port_stats.* port fdr stats get failed with error Feature unavailable.*"

Reply:
I think we have some miscommunication here:
**.* ERR syncd\d*#syncd.SAI_API_PORT:_brcm_sai_read_fec_stat_err_counters. failed with error Feature unavailable.**  should be included this PR, since it is caused by sonic-net/SONiC#1101, same with **ERR swss#orchagent: :- querySwitchLagHashAlgorithmEnumCapabilities: Failed to get attribute.**.
But I suggest don't include **. ERR syncd\d*#syncd.SAI_API_PORT:brcm_sai_get_port_stats. port fdr stats get failed with error Feature unavailable.*** in this PR, because I didn't find any of it in our nightly test.
Do you have other concerns or any misunderstanding here?

Reply:
Hi @ZhaohuiS,
My understanding is that the error for _brcm_sai_read_fec_stat_err_counters is from reading the FEC counter and unrelated to the query hash-related capacities in https://github.com/sonic-net/SONiC/pull/1101. But anyway, I could add it to this PR for the same reason of unblocking log checking. 

"port fdr stats get failed with error Feature unavailable" We saw this a lot on 7260 t0-116. We could put this in another PR in case we will need it later.